src/Instrumentation/Symfony/src/SymfonyInstrumentation.php

```diff
@@ -117,6 +117,12 @@ public static function register(): void
 
                 $span->setAttribute(TraceAttributes::HTTP_RESPONSE_BODY_SIZE, $contentLength);
 
+                // Propagate server-timing header to response, if ServerTimingPropagator is present
```

Comment:
I think that if these two propagators globally register themselves (see B3 and cloudtrace in core repo), then they will be available via `Globals::propagator()` and can do all propagation in one hit. It should also then just work with any future propagators.

Reply:
I'm wrong here, since response propagators are similar but applied differently, they will likely need their own interface in future.